scripts/build_container.sh

```diff
@@ -0,0 +1,21 @@
+#!/bin/bash
+set -e -o pipefail
+
+MUNGED_BRANCH=$(echo $CIRCLE_BRANCH | tr '/' '_')
+docker build . \
+  -t quay.io/aipg/${CIRCLE_PROJECT_REPONAME}:${CIRCLE_SHA1} \
+  -t quay.io/aipg/${CIRCLE_PROJECT_REPONAME}:${MUNGED_BRANCH} \
+  --build-arg BUILD_DATE=$(date -u +"%Y-%m-%dT%H:%M:%SZ") \
+  --build-arg GIT_SHA1=${CIRCLE_SHA1} \
+  --build-arg GIT_TAG=${CIRCLE_TAG}
+
+docker login -u "${QUAY_USER}" -p "${QUAY_PASS}" quay.io
+
+if [[ -n "${CIRCLE_TAG}" ]]; then
+  docker quay.io/aipg/${CIRCLE_PROJECT_REPONAME}:${MUNGED_BRANCH} quay.io/aipg/${CIRCLE_PROJECT_REPONAME}:${CIRCLE_TAG}
```

Comment:
We had a mistake here that was never exercised because it never ran for tags, it should be `docker tag ...`